openfl-workspace/tf_cnn_histology/src/taskrunner.py

```diff
@@ -17,40 +19,47 @@ class TensorFlowCNN(KerasTaskRunner):
     """
 
     def __init__(self, **kwargs):
-        """Initialize.
+        """
+        Initialize.
 
         Args:
             **kwargs: Additional parameters to pass to the function
 
         """
         super().__init__(**kwargs)
 
-        self.model = self.create_model(
+        self.model = self.build_model(
             self.feature_shape,
             self.data_loader.num_classes,
             **kwargs
         )
         self.initialize_tensorkeys_for_functions()
 
-    def create_model(self,
-                     input_shape,
-                     num_classes,
-                     training_smoothing=32.0,
-                     validation_smoothing=1.0,
-                     **kwargs):
-        """Create the TensorFlow CNN Histology model.
+        self.model.summary(print_fn=self.logger.info)
+
+        self.logger.info(f'Train Set Size : {self.get_train_data_size()}')
+        self.logger.info(f'Valid Set Size : {self.get_valid_data_size()}')
+
+    def build_model(self,
+                    input_shape,
+                    num_classes,
+                    **kwargs):
+        """
+        Define the model architecture.
 
         Args:
-            training_smoothing (float): (Default=32.0)
-            validation_smoothing (float): (Default=1.0)
+            input_shape (numpy.ndarray): The shape of the data
+            num_classes (int): The number of classes of the dataset
             **kwargs: Additional parameters to pass to the function
 
+        Returns:
+            keras.src.engine.functional.Functional
+
         """
         print(tf.config.threading.get_intra_op_parallelism_threads())
         print(tf.config.threading.get_inter_op_parallelism_threads())
```

Comment:
These two lines are neither relevant to the example nor OpenFL. Can be removed to reduce cognitive overhead.

Reply:
Agreed. This was copied over from an older code. Removed